app/repositories/slick/implementations/SlickChatsRepository.scala

```diff
@@ -289,13 +289,29 @@ class SlickChatsRepository @Inject() (db: Database)(implicit executionContext: E
   def moveChatToTrash(chatId: String, userId: String): Future[Boolean] =
     db.run(moveChatToTrashAction(chatId, userId))
 
+  private[implementations] def getEmailAction(chatId: String, emailId: String, userId: String) = {
+    for {
+      optionChat <- getChatAction(chatId, userId)
+    } yield optionChat match {
```

Comment:
This is a convoluted way of doing flatMap.
The for loop with just one line is unnecessary.

Reply:
Okay, I'll change that